### PR TITLE
10.0 trp project scrum with additions rebased

### PR DESCRIPTION
**Pull request by gfcapalbo**

### CODE REVIEW

File: trp_project_scrum/security/res_groups.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="external_scrum_users" model="res.groups">
```
Comment (hbrunn) [inline trp_project_scrum/security/res_groups.xml:4]: please add both of those to the external users category as in https://github.com/Therp/Therp-Addons/blob/10.0/trp_external_user/security/res_groups.xml#L5

File: trp_project_scrum/security/ir_rules.xml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        <field name="model_id" ref="base.model_ir_sequence"/>
+        <field name="domain_force">[('code', 'in', ['project.task', 'user.story'])]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_read" eval="True" />
+    </record>
+
+    <record id="rule_ir_attachments_external_scrum_users" model="ir.rule">
+        <field name="name">Read and create attachments that are on model project, task or user story</field>
+        <field name="model_id" ref="base.model_ir_attachment"/>
+        <field name="domain_force">[('res_model', 'in', ['project.project', 'user.story' , 'project.task'])]</field>
```
Comment (hbrunn) [inline trp_project_scrum/security/ir_rules.xml]: this has unexpected side effects. Check out how I changed the rule locally to make this work (js+css is cached as attachment too)

**Review by hbrunn — CHANGES_REQUESTED** [review]

File: trp_project_scrum/security/ir_rules.xml (hunk 2 above)
Comment (hbrunn) [reply trp_project_scrum/security/ir_rules.xml]: actually, reading the v10 implementation of https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_attachment.py#L356 we don't need this rule after all it seems

**Comment by gfcapalbo** [comment]
@hbrunn checks passing and all requests addressed

**Comment by hbrunn** [comment]
great. Please squash commits (only the ones not already in 8.0), then I'll merge

**Review by hbrunn — APPROVED** [review]

**Pull request merged**